### PR TITLE
Add an AutoFlip demo with improved UI.

### PR DESCRIPTION
**Pull request by WJ0417**
Please be free to leave comments.

### CODE REVIEW

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_api.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+/**
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_api.js:1]: Delete this file

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_bin.MF [hunk 2]
```diff
@@ -0,0 +1,4 @@
+javascript/closure/base.js
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_bin.MF:1]: Delete this file.

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
+.abs {
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo.css:1]: delete this file

File: web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+            `IndexDB:frame(${key + i}) couldn't be found in your database!`,
+          );
+        }
+      };
+    }
+  });
+}
+
+/** Transfers the crop windows rectangles from stream. */
+function convertSeralizedRectToObj(protoArray: number[]): Rect | undefined {
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts:286]: Change to more readable versions

Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts:286]: Fixed.

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo.css (hunk 3 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo.css]: Fixed

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_bin.MF (hunk 2 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_bin.MF]: Fixed.

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_api.js (hunk 1 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/default_demo_api.js]: Fixed

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/autoflip_live.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/autoflip_live.html:1]: I think you can delete this file

File: web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+    currentId += resultCropInfo.length;
+    resultCropInfo = [];
+    resultShots = [];
+  } else {
+    console.log(`AUTOFLIP: runTillIdle is not avaiable`);
+  }
+}
+
+/** Reads frame decode datarows from the indexDB. */
+async function read(videoId: number, start: number): Promise<Frame[]> {
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts:249]: Maybe use a more descriptive function name, like readFramesFromIndexedDb?

File: web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/autoflip_live.html (hunk 5 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_wasm/autoflip_live.html]: fixed.

File: web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts (hunk 6 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts]: Totally agree, change some function names. Please indicate more if you see any name improper.

File: web_autoflip_demo/autoflip_demo_UI/render.js [hunk 7]
```diff
@@ -0,0 +1,222 @@
+"use strict";
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/render.js:1]: Remove file

File: web_autoflip_demo/autoflip_demo_UI/render.ts [hunk 8]
```diff
@@ -0,0 +1,253 @@
+$('.custom-video-area').each(function () {
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_demo_UI/render.ts:1]: Add license

Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/render.ts:16]: Fixed.

File: web_autoflip_demo/autoflip_demo_UI/render.js (hunk 7 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_demo_UI/render.js]: Fixed.

**Comment by WJ0417** [comment]
Hi Justin, I fixed the comments before and add some new code for the `user input aspect ratio`. Please check.

File: web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts (hunk 6 above)
Comment (parrajustin) [reply web_autoflip_demo/autoflip_demo_UI/autoflip_worker.ts]: Great work!

**Comment by parrajustin** [comment]
Looks good

**Pull request merged**